src/LOAuth.ts

```diff
@@ -300,9 +300,10 @@ class LOAuth {
       } else {
         console.warn(error, 'Error refreshing access token - redirecting');
         if (this.clientOptions.loginRedirectUri) {
+          const hasQueryParams = this.clientOptions.loginRedirectUri?.split('?')[1];
           window.location.href = `${
-            this.clientOptions.loginRedirectUri
-          }?originalUrl=${encodeURIComponent(window.location.href)}`;
+            this.clientOptions.loginRedirectUri + (hasQueryParams ? '&' : '?')
+          }originalUrl=${encodeURIComponent(window.location.href)}`;
```

Comment:
It seems like this works, but maybe doing something like https://stackoverflow.com/questions/486896/adding-a-parameter-to-the-url-with-javascript might be cleaner?

Reply:
<img width="784" alt="Screen Shot 2021-12-17 at 4 54 34 PM" src="https://user-images.githubusercontent.com/9114987/146612433-609871fc-2b79-4a00-b3a7-ad01512df64f.png">